Gems/ROS2/Code/Source/RobotImporter/FixURDF/FixURDF.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/XML/rapidxml.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/string/string.h>
+
+namespace ROS2
+{
+    namespace Utils
+    {
+        //! Modifies a parsed URDF in memory to add missing inertia to links, which prevents SDF error 19.
+        //! @param urdf URDF to modify.
+        //! @returns a list of names of links that were modified.
+        AZStd::vector<AZStd::string> AddMissingInertiaToLinks(AZ::rapidxml::xml_node<>* urdf);
+
+        //! Modifies names of links and joints to be unique, which means that a set of both contains no duplicates. This prevents SDF
+        //! error 2.
+        //! @param urdf URDF to modify.
+        //! @returns a list of links that were modified
+        AZStd::vector<AZStd::string> ChangeDuplications(AZ::rapidxml::xml_node<>* urdf);
+
```

Comment:
Should these definitions get removed from the header, and then just move the comments into the cpp? I can't see why these need to be separately exposed publicly. Also, if you did that, you could move the rapidxml include into the cpp.

Reply:
It was removed from the header. Thanks for the suggestion.